src/nvim/buffer.c

```diff
@@ -931,6 +935,14 @@ static void free_buffer_stuff(buf_T *buf, int free_flags)
 }
 
 /// Go to another buffer.  Handles the result of the ATTENTION dialog.
+///
+/// @param start  Can be the following values:
+/// DOBUF_CURRENT   go to "count" buffer from current buffer
+/// DOBUF_FIRST     go to "count" buffer from first buffer
+/// DOBUF_LAST      go to "count" buffer from last buffer
+/// DOBUF_MOD       go to "count" modified buffer from current buffer
```

Comment:
this is repeating info at `do_buffer`, not to mention the enum itself: `dobuf_action_values`

if the `int` param type can't be changed to an enum type, then just link to `dobuf_action_values` from the docstring. repeating enum value docs in mutliple is not maintainable.